plugins/offboard/offboard.h

```diff
@@ -6,12 +6,27 @@ namespace dronecore {
 
 class OffboardImpl;
 
+/**
+ * The Offboard class allows do offboard velocity control of a drone.
```

Comment:
`The Offboard class enables offboard velocity control of a drone.`
- Are we sticking with drone, or are we going along with what appears to be current flavour of the month "UAS"
- Can we assume everyone coming here knows what "offboard control" means?

This is missing a word

Reply:
> Are we sticking with drone, or are we going along with what appears to be current flavour of the month "UAS"

I think drone should be ok since this is DroneCore :)

> Can we assume everyone coming here knows what "offboard control" means?

No, good point. It's very strange naming so we should at least explain it.
Let me give it a try: 2b3eb0d